implementations/rust/ockam/ockam_command/src/command/channel.rs

```diff
@@ -0,0 +1,57 @@
+use crate::service::echoer::ECHOER_SERVICE_NAME;
+use crate::AppError;
+use ockam::{
+    route, Context, Entity, Identity, Result, SoftwareVault, TcpTransport, TrustEveryonePolicy,
+    VaultSync, TCP,
+};
+use ockam_core::vault::{SecretAttributes, SecretPersistence, SecretType, SecretVault};
+
+pub struct ChannelCommand {}
+
+impl ChannelCommand {
+    pub async fn run(
+        ctx: &mut Context,
+        secret_key_path: String,
+        channel_address: &str,
+        channel_name: &str,
+        message: &str,
+    ) -> Result<(), AppError> {
+        let _tcp = TcpTransport::create(ctx).await?;
+
+        let mut vault = VaultSync::create(ctx, SoftwareVault::default()).await?;
+        let vault_address = vault.address();
+        let mut alice = Entity::create(ctx, &vault_address).await?;
+
+        let secret_key = std::fs::read_to_string(secret_key_path).unwrap();
```

Comment:
We should probably try to return an error if the path provided doesn't exist, rather than panicking.